macstadium-shared-2/Makefile

```diff
@@ -0,0 +1,61 @@
+include $(shell git rev-parse --show-toplevel)/terraform.mk
+
+.PHONY: default
+default: hello
+
+CONFIG_FILES := \
+		config/jupiter-brain-prod-org-env \
+		config/jupiter-brain-staging-org-env \
+		config/travis-worker-org-staging-1 \
+		config/travis-worker-org-staging-2 \
+		config/travis-worker-org-prod-1 \
+		config/travis-worker-org-prod-2 \
+		config/vsphere-janitor-prod-com \
+		config/vsphere-janitor-staging-com \
+		config/jupiter-brain-prod-com-env \
+		config/jupiter-brain-staging-com-env \
+		config/travis-worker-com-staging-1 \
+		config/travis-worker-com-prod-1 \
+		config/collectd-vsphere-common-1 \
+		config/travis-vm-ssh-key
+
+.PHONY: .config
+.config: $(CONFIG_FILES) $(ENV_NAME).tfvars
+
+$(CONFIG_FILES):
+	mkdir -p config
+	cp -v $$TRAVIS_KEYCHAIN_DIR/travis-keychain/macstadium/travis-vm-ssh-key config/
+	trvs generate-config -p JUPITER_BRAIN -f env jupiter-brain production-1 \
+		| sed 's/^/export /' >config/jupiter-brain-prod-org-env
+	trvs generate-config -n -p JUPITER_BRAIN -f env jupiter-brain staging-1 \
+		| sed 's/^/export /' >config/jupiter-brain-staging-org-env
+	trvs generate-config -n -p TRAVIS_WORKER -f env macstadium-workers staging-1-1 \
```

Comment:
This (and all the `1-1` and `1-2` etc.) below it should be `2-1`, the first number is the pod.